packages/stream/solana/StreamClient.ts

```diff
@@ -890,6 +894,26 @@ export default class SolanaStreamClient extends BaseStreamClient {
     return sortedStreams.filter((stream) => stream[1].type === type);
   }
 
+  public async searchStreams(data: ISearchStreams): Promise<ProgramAccount<Stream>[]> {
+    const filters: MemcmpFilter[] = [];
+    for (const [k, v] of Object.entries(data)) {
+      if (v) {
```

Comment:
could `v` be 0 intentionally? 
is there a discriminator from `offset=0` in accounts?